README.md

```diff
@@ -1,6 +1,8 @@
 # MoveIt! Tutorials
 
-This repository is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
+This repository is automatically built by two systems:
+* The Travis build output is immediately deployed to gh-pages here: https://ros-planning.github.io/moveit_tutorials/
+* The ROS build farm output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
```

Comment:
@nbbrooks please explain that 

* The ROS build farm is deprecated because it takes 2 days to build, but that it will still be used for old versions of the tutorials (e.g. indigo)